ergo-wallet/src/test/scala/org/ergoplatform/wallet/secrets/DerivationPathSpec.scala

```diff
@@ -19,4 +22,105 @@ class DerivationPathSpec
     }
   }
 
+  property("nextPath - new default derivation") {
+    implicit val enc = new ErgoAddressEncoder(ErgoAddressEncoder.MainnetNetworkPrefix)
+
+    // This testing pair is checked against CoinBarn and Yoroi
+    val mnemonic = "liar exercise solve delay betray sheriff method empower disease river recall vacuum"
+    val address = "9hAymcGaRfTX7bMADNdfWfk7CKzi2ZpvRBCmtEf6d92n8E26Ax7"
+
+    val seed = Mnemonic.toSeed(mnemonic, None)
+
+    val masterKey = ExtendedSecretKey.deriveMasterKey(seed)
+    val dp = DerivationPath.nextPath(IndexedSeq(masterKey)).get
```

Comment:
```suggestion
    val dp = DerivationPath.nextPath(IndexedSeq(masterKey), usePreEip3Derivation = false).get
```

Reply:
Suggest to pass this explicitly for readability

Reply:
done